src/gui/newaccountwizard/newaccountwizardcontroller.cpp

```diff
@@ -205,9 +206,9 @@ void NewAccountWizardController::onPageChanged(int newPageIndex)
     if (newPageIndex == _urlPageIndex) {
         _wizard->setButtonLayout(_buttonLayouts[_urlPageIndex]);
         _wizard->setButtonText(QWizard::WizardButton::NextButton, tr("Sign in"));
+        _wizard->setButtonText(QWizard::WizardButton::NextButton, tr("Open sign in again"));
     } else if (newPageIndex == _oauthPageIndex) {
         _wizard->setButtonLayout(_buttonLayouts[_oauthPageIndex]);
-        _wizard->setButtonText(QWizard::WizardButton::NextButton, tr("Open sign in again"));
```

Comment:
hmm, just confused. the following lines are setting text to the same button. looks like an override.
```cpp
        _wizard->setButtonText(QWizard::WizardButton::NextButton, tr("Sign in"));
        _wizard->setButtonText(QWizard::WizardButton::NextButton, tr("Open sign in again"));
```

Reply:
is it necessary, or can we just revert the changes?